From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Mio <mio@mio19.uk>
Date: Sat, 24 Oct 2026 15:44:37 +1100
Subject: [PATCH] devices/fossil-hoki-mainline: inline the family import

Keep the standalone mainline target's only import on one line; this is the nixfmt-stable form and makes the no-fossil-hoki guard unambiguous.
---
 devices/fossil-hoki-mainline/default.nix | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/devices/fossil-hoki-mainline/default.nix b/devices/fossil-hoki-mainline/default.nix
index 14ea85ed..a760ddaa 100644
--- a/devices/fossil-hoki-mainline/default.nix
+++ b/devices/fossil-hoki-mainline/default.nix
@@ -17,9 +17,7 @@ let
   hokiDts = ./sda429-hoki.dts;
 in
 {
-  imports = [
-    ../families/asteroidos.nix
-  ];
+  imports = [ ../families/asteroidos.nix ];
 
   nixpkgs.hostPlatform = "armv7l-linux";
 
